packages/core/src/components/templates/ProductListingPage/ProductListing.tsx

```diff
@@ -39,21 +46,31 @@ const overwriteMerge = (_: any[], sourceArray: any[]) => sourceArray
 export default function ProductListing({
   page: { sections, settings },
   data: server,
+  serverManyProductsVariables,
   globalSections,
 }: ProductListingPageProps) {
-  const {
-    state: { sort, term, selectedFacets },
-  } = useSearch()
+  const { state } = useSearch()
+  const { sort, term, selectedFacets } = state
+
   const itemsPerPage = settings?.productGallery?.itemsPerPage ?? ITEMS_PER_PAGE
 
+  const applySearchState = useApplySearchState()
+  useEffect(() => {
+    applySearchState(formatSearchState(state))
+  }, [])
```

Comment:
Updates the URL with the SearchState, which already starts with fuzzy and operator.